src/usecase/tests/order_ucase_test.go

```diff
@@ -0,0 +1,125 @@
+package tests
+
+import (
+	"github.com/Aiscom-LLC/meals-api/src/delivery"
+	"github.com/Aiscom-LLC/meals-api/src/delivery/middleware"
+	"github.com/Aiscom-LLC/meals-api/src/repository"
+	"github.com/appleboy/gofight/v2"
+	"github.com/buger/jsonparser"
+	"github.com/go-playground/assert/v2"
+	uuid "github.com/satori/go.uuid"
+	"net/http"
+	"testing"
+)
+
+func TestAddOrder(t *testing.T) {
+	r := gofight.New()
+
+	type Dish struct {
+		Amount int    `json:"amount"`
+		DishId string `json:"dishId"`
+	}
+	type Order struct {
+		Comment string `json:"comment"`
+		Items   []Dish `json:"items"`
+	}
+
+	userRepo := repository.NewUserRepo()
+	cateringRepo := repository.NewCateringRepo()
+	categoryRepo := repository.NewCategoryRepo()
+	cateringResult, _ := cateringRepo.GetByKey("name", "Twiist")
+	cateringID := cateringResult.ID.String()
+	categoryResult, _ := categoryRepo.GetByKey("name", "гарнир", cateringID)
+	categoryID := categoryResult.ID.String()
+	userResult, _ := userRepo.GetByKey("email", "admin@meals.com")
```

Comment:
same thing here about user JWT, please generate JWT for user with this email `user1@meals.com`